File: passbook-management-api/src/main/java/com/uit/passbook_management_api/dto/request/RutTienRequest.java
package com.uit.passbook_management_api.dto.request;

import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDate;

@Data
public class RutTienRequest {
    private String maSoTietKiem;
    private BigDecimal soTienRut;
    private LocalDate ngayRut;
}

File: frontend-csharp/Models/GiaoDichModel/RutTienRequest.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace frontend_csharp.Models.GiaoDichModel
{
    public class RutTienRequest
    {
        [JsonPropertyName("maSoTietKiem")]
        public string MaSoTietKiem { get; set; }

        // Dùng decimal để tránh sai số tiền tệ khi xử lý tất toán/rút gốc
        [JsonPropertyName("soTienRut")]
        public decimal SoTienRut { get; set; }

        // LocalDate ở Java tương ứng với DateTime trong C#
        [JsonPropertyName("ngayRut")]
        public DateTime NgayRut { get; set; }
    }
}
